packages/elements/src/autosuggest/__test__/autosuggest-functional.test.js

```diff
@@ -18,8 +18,8 @@ import {
   tapAction
 } from './helpers/helpers.js';
 
-describe('autosuggest/Functional', () => {
-  describe('Functionality', () => {
+describe('autosuggest/Functional', function () {
+  describe('Functionality', function () {
```

Comment:
You forget to remove arrow function in line 81

Reply:
somehow eslint couldn't detect this! Fixed.